### ISSUE TITLE
plot_model(model, plot='forecast') error in pycaret.time_series [NEW BUG]

### ISSUE THREAD

**Issue by moezali1** [pycaret/pycaret#1643]
2/10/2021.

New bug report:

```
from pycaret.datasets import get_data
data = get_data('airline')

from pycaret.time_series import *
s = setup(data, fold = 5, fh = 12, session_id = 123)

arima = create_model('arima')

plot_model(arima, plot = 'forecast')
```

![image](https://user-images.githubusercontent.com/54699234/135716199-534ace3c-4b37-488f-9327-0947465f4342.png)

`plot_model` command fails:

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_58128/3494321475.py in <module>
----> 1 plot_model(arima, plot = 'forecast')

~\pycaret\pycaret\internal\utils.py in wrapper(*args, **kwargs)
    779                 if globals_d[name] is None:
    780                     raise ValueError(message)
--> 781             return func(*args, **kwargs)
    782 
    783         return wrapper

~\pycaret\pycaret\time_series.py in plot_model(estimator, plot, return_data, display_format, data_kwargs, fig_kwargs)
   1002     system = system == "0"
   1003 
-> 1004     return _CURRENT_EXPERIMENT.plot_model(
   1005         estimator=estimator,
   1006         plot=plot,

~\pycaret\pycaret\internal\pycaret_experiment\time_series_experiment.py in plot_model(self, estimator, plot, return_data, display_format, data_kwargs, fig_kwargs, system)
   2505                         return False
   2506 
-> 2507                 prediction_interval_flag = _check_estimator_has_intervals(estimator)
   2508 
   2509                 predictions = self.predict_model(

~\pycaret\pycaret\internal\pycaret_experiment\time_series_experiment.py in _check_estimator_has_intervals(estimator)
   2499 
   2500                 def _check_estimator_has_intervals(estimator):
-> 2501                     tags = estimator.get_tags()
   2502                     if tags["capability:pred_int"]:
   2503                         return True

AttributeError: 'ARIMA' object has no attribute 'get_tags'

I have tried it with other estimators. All of them gives the same error message i.e. `EstimatorName` object has no attribute 'get_tags'.

**Comment by ngupta23** [comment]
Please upgrade to sktime 0.8.0 for this to work.

**Comment by moezali1** [comment]
This issue is resolved after sktime is updated to 0.8.
